src/main/java/core/basesyntax/service/implementation/FruitStoreServiceImplementation.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.service.FruitStoreService;
+import core.basesyntax.service.ReadFromFileService;
+import core.basesyntax.service.WriteToFileService;
+import core.basesyntax.strategy.OperationStrategy;
+
+public class FruitStoreServiceImplementation implements FruitStoreService {
+    private OperationStrategy operationStrategy;
+    private ReadFromFileService readFromFileService;
+    private CreateReportService createReportService;
+    private WriteToFileService writeToFileService;

```

Comment:
As far as I can see, you don't use these operations inside this class. Why are they here for you? In general, there is no need for such a service. It is enough just to get the data in the main, then create a report, and then write it down.